lib/Controller/StackApiController.php

```diff
@@ -39,18 +39,20 @@
  * @package OCA\Deck\Controller
  */
 class StackApiController extends ApiController {
-	private $boardService;
-	private $stackService;
 
 	/**
 	 * @param string $appName
 	 * @param IRequest $request
 	 * @param StackService $stackService
+	 * @param BoardService $boardService
 	 */
-	public function __construct($appName, IRequest $request, StackService $stackService, BoardService $boardService) {
+	public function __construct(
+		$appName,
+		IRequest $request,
+		private StackService $stackService,
+		private BoardService $boardService
+		) {
```

Comment:
I'd personally prefer to not indent the closing round brackets, but since the php-cs check passes I'm also fine with that.

Reply:
For me personally, It would be fine either way. I'm just always looking for consistency. I saw other controllers indented this way, and I just wanted to unite them.

Reply:
Then let's keep it as it is for now